lib/index.js

```diff
@@ -210,7 +199,7 @@ exports.insert = brCallbackify(async ({actor, account, meta}) => {
  *
  * @returns {Promise} Resolves to a boolean indicating account existence.
  */
-exports.exists = brCallbackify(async (
+exports.exists = async (
   {actor, id, email, status = 'active'}) => {
```

Comment:
My personal style preference here is this.  This is because it makes the indent on the following line up in a more readable way.
```suggestion
exports.exists = async ({
  actor, id, email, status = 'active'
}) => {
```